oemof/solph/custom.py

```diff
@@ -797,3 +798,121 @@ def _relation_rule(block, n, t):
             return expr == 0
         self.relation = Constraint(self.OFFSETTRANSFORMERS, m.TIMESTEPS,
                                    rule=_relation_rule)
+
+class PiecewiseLinearTransformer(Transformer):
+    """An object with one inputs and one output.
+
+    Parameters
+    ----------
+    x : list
+    y : list
```

Comment:
The parameters are not documented properly:
* `x` might be `in_breakpoints`
* `y` might be `out_breakpoints`
* `conversion_function` remains fully unclear
* `pw_repn` remains fully unclear